build/yamls/antrea-migrator.yml

```diff
@@ -0,0 +1,87 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: antrea-migrator
+  namespace: kube-system
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-migrator
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-migrator
+  namespace: kube-system
+subjects:
+  - kind: ServiceAccount
+    name: antrea-migrator
+    namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: antrea-migrator
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
I think these are not required since it doesn't need any permission. It could leave the serviceAccount as empty to use the default one?

Reply:
Removed RBAC definition and set the ServiceAccount of DS to default.